### PR TITLE
-fix wrong param name

### PR DESCRIPTION
**Pull request by sbenedicadb**

### CODE REVIEW

File: AEPCore/Sources/core/MobileCore.swift [hunk 1]
```diff
@@ -95,8 +95,8 @@ public final class MobileCore: NSObject {
     /// Submits a generic event containing the provided IDFA with event type `generic.identity`.
     /// - Parameter identifier: the advertising identifier string.
     @objc(setAdvertisingIdentifier:)
-    public static func setAdvertisingIdentifier(adId: String?) {
```
Comment (nporter-adbe) [inline AEPCore/Sources/core/MobileCore.swift:98]: I'd prefer to change the doc comment than the code, as it could break sample apps.

Comment (sbenedicadb) [reply AEPCore/Sources/core/MobileCore.swift:98]: it does break sample code (and tests...), but i think `identifier` is a more correct name for the parameter than `adId`.  thoughts?

Comment (nporter-adbe) [reply AEPCore/Sources/core/MobileCore.swift:98]: don't have much of a preference, I think they're about equal in my book. up to you.

**Comment by sbenedicadb** [comment]
closing for deeper review of public apis

**Pull request closed without merging**